### PR TITLE
Update ads/README.md with gulp lint tip

### PR DESCRIPTION
**Pull request by lannka**

### CODE REVIEW

File: ads/README.md [hunk 1]
```diff
@@ -248,3 +248,4 @@ If you have non-trivial logic in `/ads/yournetwork.js`, adding a unit test at `/
 
 - Please consider implementing the `render-start` and `no-content-available` APIs (see [Available APIs](#available-apis)), which helps AMP to provide user a much better ad loading experience.
 - [CLA](../CONTRIBUTIONG.md#contributing-code): for anyone who has trouble to pass the automatic CLA check in a pull request, try to follow the guidelines provided by the CLA Bot. Common mistakes are 1) used a different email address in git commit; 2) didn't provide the exact company name in the PR thread. 
+- Run `gulp lint` and fix errors if any before sending out the PR.
```
Comment (cramforce) [inline ads/README.md]: Also `gulp check-types`

Comment (lannka) [inline ads/README.md]: Did we already enabled check-types for 3p? I checked gulpfile seems not yet

**Comment by cramforce** [comment]
I accidentally deleted it :) Lets add it to the doc.

**Comment by lannka** [comment]
Added

**Pull request merged**